### ISSUE TITLE
How does "it will adjust itself" work?

### ISSUE THREAD

**Issue by lostmsu** [joshferns/KinectHolographic#2]
Hi!

I hope 4 years later you still have enough interest in this project to answer a question!

TL;DR; the question is how does the camera "adjusts itself" once you supply it with a head position information in some arbitrary 3D coordinate system?

#### A longer form

I arrived here after scratching my head a lot (I have nearly 0 experience in 3D graphics geometry), after trying to make basically the same thing for fun, but based on [FaceMesh](https://github.com/tensorflow/tfjs-models/tree/master/facemesh), which is a neural network, that does accurate head mesh position estimation from webcam image.

The problem I faced is: given the head position in FaceMesh camera 3D coordinates (e.g. colorX, colorY, depth), how do I get the location of the physical screen in the same coordinates to be able to build a projection matrix for my virtual camera, representing the eyes.

Reading README and code here got me to understand, that I need the "Generalized Perspective Projection". I think it is pretty clear, that it is exactly the projection I need. However! I am grokking, that it requires you to know the position of the screen in the same real world coordinates I use for the head, but for the love of God I could not understand how the Hell does the camera in your example "adjusts itself" instead of you manually computing the location of the screen based on some reference points.

To be clear, I have not run this example, so might misunderstand how it works, but you did not show any manual adjustment in the video, so I assumed it does indeed "autoadjusts" somehow.
